fedora_elections/elections.py

```diff
@@ -125,10 +122,11 @@ def vote(election_alias):
             'Unknown election voting type: %s' % election.voting_type)
         return safe_redirect_back()
 
-
-def vote_range(election):
+def vote_range(election, revote):
     votes = models.Vote.of_user_on_election(
-        SESSION, flask.g.fas_user.username, election.id, count=True)
+        SESSION, flask.g.fas_user.username, election.id)
+
+    sorted_votes = sorted(votes, key=lambda candidate: candidate.candidate_id)
```

Comment:
Is this still needed since the Vote.of_user_on_election is now ordered by candidate_id ?